scripts/mkcloudhost/cloudfunc

```diff
@@ -45,3 +45,10 @@ function routedcloudpublicnet()
     base2=$(($base2 % 256))
     echo -n 10.$base1.$(($base2+$n-1))
 }
+
+function ipv6prefix()
+{
+    local n=$1
+    local mkch_id=$(mkch_id)
+    printf fd3b:f495:9651:%x%02x "$mkch_id" "$n"
```

Comment:
Cool, went and generated one for each letter. We could have up to 256 mkcloud hosts (if mkch_id supported more then 1 letter of the alphabet) and to keep it encoded into a valid ipv6 address. and obviously many more slots then the current 9 :)

That should do for now :P